src/core/MOM_open_boundary.F90

```diff
@@ -203,8 +205,8 @@ module MOM_open_boundary
   real, allocatable :: Cg(:,:)  !< The external gravity wave speed [L T-1 ~> m s-1]
                                 !! at OBC-points.
   real, allocatable :: Htot(:,:)  !< The total column thickness [H ~> m or kg m-2] at OBC-points.
-  real, allocatable :: dZtot(:,:) !< The total column vertical extent [Z ~> m] at OBC-points.
-  real, allocatable :: h(:,:,:)   !< The cell thickness [H ~> m or kg m-2] at OBC-points.
+  real, allocatable :: dZtot(:,:) !< The total column vertical extent [Z ~> m] at OBC segment faces.  This
```

Comment:
Trailing "This" is out of place.

Reply:
Thanks for spotting this "This". It has now been removed.